addons/sale_management/models/sale_order.py

```diff
@@ -152,7 +152,7 @@ def action_confirm(self):
         res = super(SaleOrder, self).action_confirm()
         for order in self:
             if order.sale_order_template_id and order.sale_order_template_id.mail_template_id:
-                self.sale_order_template_id.mail_template_id.send_mail(order.id)
+                self.with_user(SUPERUSER_ID).sale_order_template_id.mail_template_id.send_mail(order.id)
```

Comment:
```suggestion
                self.sale_order_template_id.mail_template_id.with_user(SUPERUSER_ID).send_mail(order.id)
```
If the user change is only necessary for the send_mail part, only put it where it's really needed :)

Reply:
:ok_hand: